man/man1/git-secret-hide.1.ronn

```diff
@@ -3,14 +3,33 @@ git-secret-hide - encrypts all added files with the inner keyring.
 
 ## SYNOPSIS
 
-    git secret hide [-c] [-P] [-v] [-d] [-m]
+    git secret hide [-c] [-F] [-P] [-v] [-d] [-m]
 
 
 ## DESCRIPTION
-`git-secret-hide` creates an encrypted version (typically called filename.txt.secret) 
-for each file added by `git-secret-add` command. 
+`git-secret-hide` creates an encrypted version (typically called `filename.txt.secret`) 
+of each file added by `git-secret-add` command. 
 Now anyone enabled via 'git secret tell' can can decrypt these files. Under the hood,
-`git-secret` uses the keyring in .gitsecret/keys and their secret key to decrypt the files.
+`git-secret` uses the keyring in `.gitsecret/keys` and user's secret keys to decrypt the files.
+
+Because a repository of secret files should be accessible by all the users listed
+in the repo's keychain (as shown by `git secret whoknows`), it is recommended that you
+use `hide` to encrypt (and if needed, re-encrypt) all the files in a git-secret repository 
+when using `hide`.
```

Comment:
Sorry, but I don't get this one: 

> Because a repository of secret files should be accessible by all the users listed in the repo's keychain (as shown by `git secret whoknows`), it is recommended that you use `hide` to encrypt (and if needed, re-encrypt) all the files in a git-secret repository when using `hide`.

Especially this part:

> it is recommended that you use `hide` to encrypt (and if needed, re-encrypt) all the files in a git-secret repository when using `hide`

Could we please rephrase it?

Reply:
rephrased!